### PR TITLE
Fix/action required

### PR DESCRIPTION
**Pull request by leifermendez**
# What is it?
New  feature for notice ACTION REQUIRED
https://share.vidyard.com/watch/p9SQkA154RQNdtMJj4dr7M
image

- [ x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Example: After `npm run qwik add tailwind` the user need to add lines into global.css file 

# Checklist:

- [x ] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [ x] I have performed a self-review of my own code
- [x ] I have made corresponding changes to the documentation
- [x ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: starters/features/tailwind/package.json [hunk 1]
```diff
@@ -2,8 +2,18 @@
   "description": "Use Tailwind into your Qwik app",
   "devDependencies": {
     "postcss": "^8.4.16",
-    "tailwindcss": "^3.1.8"
+    "tailwindcss": "^3.1.8",
+    "autoprefixer": "10.4.11"
   },
+  "nextSteps": [
```
Comment (manucorporat) [inline starters/features/tailwind/package.json:8]: let's add this to __qwik__, we have all this kind of functionality there

Comment (leifermendez) [reply starters/features/tailwind/package.json:8]: ok! mmm what do you mean ? Do I need to move this?

Comment (manucorporat) [reply starters/features/tailwind/package.json:8]: inside:
`__qwik__`

**Comment by manucorporat** [comment]
Amazing work Leifer! as usual!!

**Pull request merged**